src/Accounts/Accounts/Models/RMProfileClient.cs

```diff
@@ -111,6 +111,11 @@ public bool TryRemoveContext(IAzureContext context)
             return result;
         }
 
+        private IAzureTenant GetDetailedTenantFromQueryHistory(List<AzureTenant> tenants, string tenantId)
```

Comment:
What does history mean? Do you mean cache?

Reply:
Yes, if we queried tenants in L244
```
tenants = ListAccountTenants(account, environment, password, promptBehavior, promptAction).ToList();
```